cloud/client.go

```diff
@@ -1382,3 +1383,7 @@ func (c *client) UploadLog(pathOnDisk string) (string, error) {
 
 	return fmt.Sprintf(uploadBundleResponse.ViewURL), nil
 }
+
+func (c *client) GetAuthToken() string {
+	return c.authToken
```

Comment:
should this force a refresh if it's expired?

Reply:
We only ever call it right after newing up the client, which IIRC refreshes the token if needed.